compiler-rt/lib/asan/asan_interceptors.cpp

```diff
@@ -47,6 +48,8 @@
 #    define ASAN_PTHREAD_CREATE_VERSION "GLIBC_2.2"
 #  endif
 
+#define MAP_FIXED 0x0010 /* [MF|SHM] interpret addr exactly */
```

Comment:
`map_fixed` (in lowercase) is defined in compiler-rt/lib/sanitizer_common/sanitizer_platform_limits_posix.cpp for this purpose (example usage: compiler-rt/lib/hwasan/hwasan_interceptors.cpp).